zenhappens/src/js/rangeslider.js

```diff
@@ -0,0 +1,42 @@
+// javascript by Trevan Hetzel on codepen
+// https://codepen.io/trevanhetzel/pen/rOVrGK
+
+var sheet = document.createElement('style'),  
+  $rangeInput = $('.range input'),
+  prefs = ['webkit-slider-runnable-track', 'moz-range-track', 'ms-track'];
+
+document.body.appendChild(sheet);
+
+var getTrackStyle = function (el) {  
+  var curVal = el.value,
+      val = (curVal - 1) * 16.666666667,
+      style = '';
+  
+  // Set active label
+  $('.range-labels li').removeClass('active selected');
+  
+  var curLabel = $('.range-labels').find('li:nth-child(' + curVal + ')');
+  
+  curLabel.addClass('active selected');
+  curLabel.prevAll().addClass('selected');
+  
+  // Change background gradient
+  for (var i = 0; i < prefs.length; i++) {
+    style += '.range {background: linear-gradient(to right, #6D9E11 0%, #6D9E11 ' + val + '%, #fff ' + val + '%, #fff 100%)}';
+    style += '.range input::-' + prefs[i] + '{background: linear-gradient(to right, #6D9E11 0%, #6D9E11 ' + val + '%, #F1F1F1 ' + val + '%, #F1F1F1 100%)}';
+  }
+
+  return style;
+}
+
+$rangeInput.on('input', function () {
+  sheet.textContent = getTrackStyle(this);
```

Comment:
When the input is triggered, update the aria labels
```suggestion
  newValue = $(this).val();
  sheet.textContent = getTrackStyle(this);
  $rangeInput.attr("aria-valuenow", newValue.toString());
  $rangeInput.attr("aria-valuetext", times[newValue]);
```

Reply:
These two changes, plus two additions to the slider markup and then screenreader users will hear "15 minutes, 30 minutes" etc as they move through the slider options! 🙌